gui/index.go

```diff
@@ -127,12 +127,6 @@ func (ui *GUI) Homepage(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if len(work) == 0 && len(payments) == 0 {
```

Comment:
Why is this check no longer needed?

Reply:
I don't think this check was ever required. Removing the check has no negative consequences, but it allows users to check their connected clients or lookup their account ID regardless of whether they have any mined work or payments.

Reply:
Ah yeah this would prevent the page from loading, the use cases you've stated are valid.